pkg/kf/apps/push-options.yml

```diff
@@ -1,7 +1,7 @@
 # This file contains options for option-builder.go
 ---
 package: apps
-imports: {"io":"", "os":""}
+imports: {"io":"", "os":"", "github.com/google/kf/pkg/apis/kf/v1alpha1"}
```

Comment:
nit: Can we change this to `"github.com/google/kf/pkg/apis/kf/v1alpha1":""` to match the other imports?

Reply:
done